IdentityCore/tests/integration/MSIDHttpRequestIntegrationTests.m

```diff
@@ -108,9 +109,14 @@ - (void)testUrlRequest_byDefaultIsNil
     XCTAssertNil(self.request.urlRequest);
 }
 
-- (void)testRequest_byDefaultUseMSIDJsonResponseSerializer
+- (void)testRequest_byDefaultUseMSIDHttpResponseSerializerWithMSIDJsonResponsePreprocessor
 {
-    XCTAssertTrue([self.request.responseSerializer isKindOfClass:MSIDJsonResponseSerializer.class]);
+    XCTAssertTrue([self.request.responseSerializer isKindOfClass:MSIDHttpResponseSerializer.class]);
+    if ([self.request.responseSerializer isKindOfClass:MSIDHttpResponseSerializer.class])
```

Comment:
Curious on why we have a if here,
shouldn't this always the case as we are checking this very case?

Reply:
Removed, it is not actually needed here.